docs/take-to-prod.md

```diff
@@ -0,0 +1,108 @@
+Taking your Spark .Net Application to Production
+===
+
+# Table of Contents
+This how-to provides general instructions on how to take your .NET for Apache Spark application to production.
+In this documentation, we will summary the most commonly asked scenarios when running Spark .Net Application.
+And you will also learn how to package your application and submit your application with [spark-submit](https://spark.apache.org/docs/latest/submitting-applications.html) and [Apachy Livy](https://livy.incubator.apache.org/).
+- [How to take your application to production when you have single dependency](#how-to-take-your-application-to-production-when-you-have-single-dependency)
+  - [Scenarios - Scenario 1 and Scenario 2](#scenarios---single-dependency)
```

Comment:
Could we potentially just say "Scenarios" (so that users don't immediately wonder what 1 and 2 mean), or include a couple words about the scenarios in this header/link (so that users know what they mean), and/or divide each scenario onto its own bullet point/line?

Reply:
I think just say `Scenarios` would be good otherwise it's gonna be very long.